### PR TITLE
Dap clean comment box after submission

### PR DESCRIPTION
**Pull request by sjlarson92**

### CODE REVIEW

File: src/DailyArtPromptApp.test.js [hunk 1]
```diff
@@ -108,30 +81,18 @@ describe('mapStateToProps', () => {
     const result = mapStateToProps(state)
     expect(result.promptsImages).toEqual('promptsImages')
   })
-
 })
 
 describe('mapDispatchToProps', () => {
```
Comment (lgc13) [inline src/DailyArtPromptApp.test.js]: I think you don't have mapDispatchToProps, right>

File: src/components/Image.test.js [hunk 2]
```diff
@@ -21,10 +22,15 @@ describe('<Image />', () => {
...
 
-    it('should call onDoubleClick from props when image is doubleClicked', () => {
+    it('render with correct id prop', () => {
       const wrapper = shallow(<Image {...defaultProps}/>)
-      wrapper.find('img').simulate('doubleClick')
-      expect(defaultProps.onDoubleClick).toHaveBeenCalled()
+      expect(wrapper.find('img').prop('id')).toEqual(defaultProps.image.id)
     })
+
+    // it('should call onDoubleClick from props when image is doubleClicked', () => {
```
Comment (lgc13) [inline src/components/Image.test.js]: I think you want to test this right? (something like it at least)

File: src/components/Image.js [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
-    src={src}
-    alt={name}
-    onDoubleClick={onDoubleClick}
+    src={image.src}
+    alt={image.name}
+    onDoubleClick={() => updatePromptImages(image.id)}
   />
 
-export default Image;
+export const mapDispatchToProps = (dispatch) => ({
```
Comment (lgc13) [inline src/components/Image.js]: it doesn't look like this is being tested

File: src/components/ImageLayout.js [hunk 4]
```diff
@@ -1,17 +1,28 @@
...
+  const onChange = (e) => {
+    setInputBoxText(e.target.value)
+  }
+  const onKeyDown = (e, imageId) => {
+    if (e.keyCode === 13) {
+      const value = e.target.value
+      addComment(value, imageId)
+      setInputBoxText("")
+    }
+  }
```
Comment (lgc13) [inline src/components/ImageLayout.js]: looks like none of these are being tested?

File: src/DailyArtPromptApp.test.js (hunk 1 above)
Comment (sjlarson92) [reply src/DailyArtPromptApp.test.js]: this is true, i have deleted this test

File: src/components/Image.test.js (hunk 2 above)
Comment (sjlarson92) [reply src/components/Image.test.js]: Yes I have uncommented it :)

File: src/components/Image.js (hunk 3 above)
Comment (sjlarson92) [reply src/components/Image.js]: Yes, I have added this test

File: src/components/ImageLayout.js (hunk 4 above)
Comment (sjlarson92) [reply src/components/ImageLayout.js]: now they are :)

**Pull request merged**